app/models/mixins/miq_provision_quota_mixin.rb

```diff
@@ -250,39 +261,81 @@ def quota_find_active_prov_request_by_owner(options)
     quota_find_active_prov_request(options).select { |p| email.casecmp(p.get_option(:owner_email).to_s.strip) == 0 }
   end
 
+  def quota_find_active_prov_request_by_user(options)
+    quota_find_active_prov_request(options).where(:userid => miq_request.requester.userid)
+  end
+
   def quota_find_active_prov_request_by_group(options)
-    prov_requests = []
-    prov_owner = get_owner
-    unless prov_owner.nil?
-      group = prov_owner.ldap_group
-      prov_requests = quota_find_active_prov_request(options).select do |p|
-        prov_req_owner = p.get_owner
-        prov_req_owner && group.casecmp(prov_req_owner.ldap_group) == 0
-      end
+    prov_request_group = miq_request.requester.current_group
+    quota_find_active_prov_request(options).select do |r|
+      prov_request_group == r.requester.current_group
     end
-    prov_requests
+  end
+
+  def quota_find_active_prov_request_by_tenant(options)
+    quota_find_active_prov_request(options).where(:tenant => miq_request.tenant)
   end
 
   def quota_find_active_prov_request(_options)
     prov_req_ids = MiqQueue.where(
-      :class_name  => 'MiqProvisionRequest',
-      :method_name => 'create_provision_instances',
+      :class_name  => %w(MiqProvisionRequest ServiceTemplateProvisionRequest),
+      :method_name => 'create_request_tasks',
       :state       => 'dequeue'
     ).pluck(:instance_id)
 
     prov_ids = []
     MiqQueue
       .where(:method_name => 'deliver', :state => %w(ready dequeue), :class_name => 'MiqAeEngine')
-      .where("tracking_label like ?", '%miq_provision_%')
+      .where("task_id like ?", '%_provision_%')
```

Comment:
I think you want to continue using the `tracking_label`.  (See commit https://github.com/ManageIQ/manageiq/commit/b609e1d6d59ed9f44a2faa0eae825cc021f39afd from PR https://github.com/ManageIQ/manageiq/pull/15443)

cc @mkanoor